app/javascript/dashboard/components/widgets/forms/Input.vue

```diff
@@ -63,3 +63,21 @@ export default {
   },
 };
 </script>
+<style scoped lang="scss">
+.container {
```

Comment:
Please do not use standard names like `.container` for the class name as it would have inherited properties.